meeting_notes/2018-0-28.md

```diff
@@ -0,0 +1,52 @@
+## Participants
+
+- Alois Reitbauer
+- Bogdan Drutu
+- Sergey Kanzhelev
+- Erika Arnold
+- Ted Young
+- Reiley
+- Yuri Shkuro
+
+
+
+## Decision to make specs public
+
+Alois asked whether anyone objects making the specs public. People uniformely agreed to officially share the specs
+
+## Test Suite
+
+Reiley did a demo of the current state of the harness. 
+
+Yuri: How to control the behaviour of the harness? 
+Sergey: We will trigger different URL for different behaviour
+Yuri: Makes sense, but not currently part of PR
+
+Yuri: Can we make it simpler to only use a single language
+Reiley and Sergey: Agreed?
+
+Yuri: CLI should be primary goal as it is easy to integrate into Continuous Build systems. 
+
+Yuri: Best case we use a Docker image to run the tests. 
+
+Yuri: Did we discuss having shared libraries for parsing
+Sergey: So far we did not discuss. Opencensus can be the lib for us
+Alois: Fabian showed a library last time. 
+Bogdan: Where do we put - like an 
```

Comment:
discussion moved to github: https://github.com/w3c/distributed-tracing/issues/169